pipelines/deployment.yml

```diff
@@ -4025,22 +4016,19 @@ disabled-jobs:
         inputs:
           - name: rm-survey-service-source
         outputs:
-          - name: rm-survey-service-build
-            path: rm-survey-service
+          - name: survey-service
```

Comment:
Do the preprod and prod jobs need to exist here are the "real" ones are separate jobs now (in `pipelines/survey-service.yml `)?